### ISSUE TITLE
Re-design of Group page

### ISSUE THREAD

**Issue by tomomiq** [cobudget-org/cobudget#909]
**Describe the feature**

Objective: Move away from the "card" style on the Group page, to avoid confusion with the Round page. 

* Use the "Archive round" setting at the Round level to differentiate between Active vs Archived round 
* Re-use (as much as possible) the table-style from the Round page, grid view 
* See Round About page for having black and grey text styles in the same table row 

<img width="908" alt="Screenshot 2022-12-06 at 17 07 49" src="https://user-images.githubusercontent.com/1287179/205962962-368642c3-320c-4e98-961c-66134799b8ba.png">

<img width="211" alt="Screenshot 2022-12-06 at 17 13 12" src="https://user-images.githubusercontent.com/1287179/205964173-313be93e-e589-4770-bb66-85c511db1f30.png">

**Comment by alinauroz** [comment]
@tomomiq  where should `Todos` be placed?


<img width="1358" alt="image" src="https://user-images.githubusercontent.com/38759611/206923868-d9d878c4-bcc3-4683-a518-ab629847c16e.png">

**Comment by tomomiq** [comment]
Does this disappear when you've done all three things...? 

Let's stack it above Active Rounds (if exists) and make it the same width. 
Here's a crappy copy-paste of screenshots! Please play with the styles so that it looks clean. 

I wonder if everything under the Rounds tab being full-width might be too much white space... can you share a screenshot of what it looks like when the three elements ("Let's get this ball rolling" and two tables) have the same column width as those on Round > About pages? 

<img width="701" alt="Screenshot 2022-12-11 at 22 03 16" src="https://user-images.githubusercontent.com/1287179/206928634-b8665987-60cd-4629-ac52-164b59cc66e7.png">

**Comment by alinauroz** [comment]
No, it is not disappearing. Stuck at point 2. This might be a bug.

**Comment by alinauroz** [comment]
@tomomiq  does it look fine?

<img width="1440" alt="Screenshot 2022-12-14 at 12 49 16 AM" src="https://user-images.githubusercontent.com/38759611/207430234-7d1b2fe3-9c1e-49bf-a627-4660b2976b87.png">

**Comment by tomomiq** [comment]
The "Let's get this ball rolling" part looks great! 

* Can I see what it looks like with the group logo? 
* Remove “New round” button 
* Change tab copy from “Overview” to “Rounds” 
* Use the status label style (yellow) as the highlight style for your funds

**Comment by alinauroz** [comment]
@tomomiq 

<img width="1440" alt="Screenshot 2022-12-15 at 1 00 51 AM" src="https://user-images.githubusercontent.com/38759611/207702211-2466805d-f40f-4dcd-9b4f-429e7010a6eb.png">

**Comment by aerugo** [comment]
@tomomiq could you test this and see if it is according to your specs, it is live on staging now
